src/App.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+  import { db } from "./firebase.js";
+  export let name;
+  let arrList = [];
+  let newItemText = "";
+  let newItemQuantity;
+
+  db.collection("grocery").onSnapshot((snapData) => {
+    arrList = snapData.docs;
+  });
+
+  function addItem() {
+    db.collection("grocery").add({
+      name: newItemText,
+      quantity: newItemQuantity,
+    }); // It adds new data to the collection grocery
+    newItemText = "";
+    newItemQuantity = "";
+  }
+
+  function removeItem(itemId) {
+    console.log("The event works");
+    db.collection("grocery").doc(itemId).delete();
+  }
+</script>
+
+<main>
+  <h1>Grocery list</h1>
+  <h2>
+    <input type="text" bind:value={newItemText} placeholder="Item to buy" />
+    <input
+      type="number"
+      bind:value={newItemQuantity}
+      placeholder="Quantity to buy"
+    />
```

Comment:
This is a form, so it should be wrapped in a `<form>` element and should have labels connected.

Reference: https://www.w3.org/WAI/tutorials/forms/labels/